### DISCUSSION TITLE
"Expand folder shortcuts" not working on Win11 23H2?

### DISCUSSION THREAD

**Discussion by splisp** [Open-Shell/Open-Shell-Menu#1900] [General]
I have the "Expand folder shortcuts" option in "General behaviour" checked, but folder shortcuts do not expand.

![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2437263/376b308a-0b41-4250-a787-c59656ea136e)

Win11 Pro 23H2 build 22631.3296. I'm not on remote desktop anymore!

**Comment by splisp** [comment]
I have created a junction in the Start Menu folder resolving to the folder I want to have in the Menu (so, "C:\Users\USERNAME\AppData\Roaming\Microsoft\Windows\Start Menu\Environment" is a hardlink to "C:\WorkArea\Environment" as in my screenshot) and the item in the menu is now expanded; if I just create a link by drag & drop, it's not expanded...

**Reply by ge0rdi** [reply]
I'm unable to replicate the issue :(

For me the folder shortcut is properly expanded:
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/ed4a5040-38fb-4099-b784-b122a1036178)

What `Open-Shell` version do you have?
